include/inexor/vulkan-renderer/descriptor.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <cassert>
+#include <string>
+#include <vector>
+
+#include "spdlog/spdlog.h"
```

Comment:
We should use `<` and `>` instead of `"` for library includes.

Reply:
done.